bundles/org.openhab.binding.icalendar/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -171,7 +172,8 @@
 					<option value="WEEK">week</option>
 				</options>
 				<default>HOUR</default>
-				<label>Date or Time Unit for Start and End</label>
+				<label>Date/Time Unit</label>
+				<description>Date or Time Unit for Start and End</description>
```

Comment:
For description, use uppercase only for the first letter of the first word, not for all words.

Reply:
OK. Is there an english language based reason when to use capitalization or are labels nouns just capitalized by convention?